tests/dbz-twin/rac/soak.sh

```
@@ -0,0 +1,42 @@
+#!/bin/bash
+# Soak driver: back-to-back fuzz cycles until DURATION_SEC elapses or a cycle fails.
+# Assumes `fuzz-test.sh up` was already run.
+set -e
+cd "$(dirname "$0")"
+source /home/rophy/projects/OpenLogReplicator/tests/environments/rac/vm-env.sh
```

Shellcheck (0.11.0)

[info] Not following: /home/rophy/projects/OpenLogReplicator/tests/environments/rac/vm-env.sh was not specified as input (see shellcheck -x).

(SC1091)